tasks/packager/sectv-tizen.js

```diff
@@ -368,5 +368,40 @@ module.exports = {
                 }
             }
         });
+    },
+    deploy: function (successCallback, errorCallback, data) {   // Smart Developement Bridge
+        console.log('\nStart deploying Tizen Samsung Smart TV Platform apps......');
+
+        var { spawn } = require('child_process');
+        var wgtPath = data.wgtPath || './platform/tizen/build/app.wgt';
```

Comment:
the default dest path should be ./platform/sectv-tizen/build/app.wgt

It's sorry for saying that,
There are some parts of this PR that are questionable as to whether it works properly, so It's hard to be accepted for now.
Please test all platforms and make sure that they all work well.
Please consider to attach the working well screenshots for each platforms,
Then I will accept this PR.